.travis.yml

```diff
@@ -1 +1,24 @@
 language: go
+
+env:
+  - WITH_COVERAGE=true
+  - GOFLAGS='-race'
+
+go_import_path: github.com/google/monologue
+
+matrix:
+  fast_finish: true
+
+install:
```

Comment:
It looks like by setting an 'install' and 'script' section in this file, we don't get any of the default testing that travis can do, because it overwrites those two sections.  Is that intentional?  It makes it so that we have to specify exactly what we want to be run, and keep up with all external changes etc ourselves.  There do appear to be `before_install`, `before_script` and `after_script` sections we can add to have things run in addition to the default Travis stuff if we wanted to - could that be a better way to reap the benefits of still running all the default things Travis runs, and then adding some of our own checks on top of that?

Reply:
Yes, it's intentional.  The default stuff that Travis runs is pretty minimal; the `scripts/presubmit.sh` runs a lot more things (linters, autogeneration checks, license checks). Also, having a single self-contained script means it's easy for developers to run all the checks locally in one go.